ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # Change Log - oav
 
+## 02/24/2022 2.11.8
+
+- FIx GC introduced by 2.11.7
```

Comment:
Can you change the log if the previous implementation doesn't cause GC?

Reply:
The GC is introduced by 2.11.7, this PR will fix this.